tests/e2e_tests/Dockerfile.gui

```diff
@@ -17,7 +7,10 @@ RUN mkdir -p downloaded-tools && \
   curl -SL --fail "https://stress-client.s3-accelerate.amazonaws.com/release/mender-stress-test-client" \
   -o /usr/bin/mender-stress-test-client && \
   chmod +x /usr/bin/mender-artifact && \
-  chmod +x /usr/bin/mender-stress-test-client && \
-  ln -s /usr/bin/mender-artifact . && ln -s /usr/bin/mender-stress-test-client .
-RUN mv e2e_tests/* .
-CMD python -u test_ui.py
+  chmod +x /usr/bin/mender-stress-test-client
```

Comment:
Discussed in person: I think the mender-stress-test-client is not in use and can be removed.

As far as I can see, the tests use the Docker image `mender-client-qemu` and not the stress client